app/models/Signup.js

```diff
@@ -129,8 +131,10 @@ signupSchema.statics.lookupCurrent = function (user, campaign) {
       .then(signupDoc => resolve(signupDoc))
       .catch((err) => {
         app.locals.stathatError(statName, err);
```

Comment:
We're currently sending the status code for external network requests through `stathatError`, which help provide some insight into some of those errors we get back after broadcasts. 

@rapala61 I'm not opposed to adding the error message in the stat name as well.. but wonder if it might be redundant (as the code should always match). It could potentially catch some non-network errors like Mongo writes, but New Relic would be logging those. 

<img width="815" alt="screen shot 2017-03-22 at 8 35 58 am" src="https://cloud.githubusercontent.com/assets/1236811/24206218/cfb69e18-0eda-11e7-91ce-fc1e1a2b601d.png">